tests/components/gree/test_init.py

```diff
@@ -0,0 +1,38 @@
+"""Tests for the Gree Integration."""
+
+from homeassistant.components.gree.const import DOMAIN as GREE_DOMAIN
+from homeassistant.config_entries import ENTRY_STATE_LOADED, ENTRY_STATE_NOT_LOADED
+from homeassistant.setup import async_setup_component
+
+from tests.async_mock import patch
+from tests.common import MockConfigEntry
+
+
+async def test_setup_simple(hass):
+    """Test gree integration is setup."""
+    await async_setup_component(hass, GREE_DOMAIN, {})
+    await hass.async_block_till_done()
+
+    # No flows started
+    assert len(hass.config_entries.flow.async_progress()) == 0
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6333224)